R/check_source.R

```diff
@@ -1,5 +1,19 @@
 #' Standardised test for an R source repository
 #'
+#' A convenience function that runs all source related tests in sequence.
+#' The details lists the relevant functions.
+#' When you fixed a problem, you can speed things up by running only the related
+#' check.
+#' We still recommend to run `check_source()` before you push to GitHub.
+#' And only push when the functions indicates that there are no problems.
+#' This caches most problems before sending the code to GitHub.
```

Comment:
See remarks for the `check_package()` documentation, maybe you can just copy it after correction?